package on2013_03.on2013_03_10_Codeforces_Round__172.A___Rectangle_Puzzle;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2013_03/on2013_03_10_Codeforces_Round__172/A___Rectangle_Puzzle/A - Rectangle Puzzle.task"))
			Assert.fail();
	}
}
